azkaban-web-server/build.gradle

```diff
@@ -1,191 +1,193 @@
-  }
 }
 
 // This node plugin enables users to run any NodeJS script.  The plugin will download and
 // manage NodeJS distributions, use them from there.
 // ***npm install*** installs all dependencies in package.json. It will only run when changes are made to package.json
 plugins {
-  id "com.moowork.node" version "0.13"
+    id "com.moowork.node" version "0.13"
 }
 
 node {
-  // Version of node to use.
-  version = '6.2.1'
+    // Version of node to use.
+    version = '6.2.1'
 
-  // Version of npm to use.
-  npmVersion = '3.9.3'
+    // Version of npm to use.
+    npmVersion = '3.9.3'
 
-  // Base URL for fetching node distributions (change if you have a mirror).
-  distBaseUrl = 'https://nodejs.org/dist'
+    // Base URL for fetching node distributions (change if you have a mirror).
+    distBaseUrl = 'https://nodejs.org/dist'
 
-  // If true, it will download node using above parameters.
-  // If false, it will try to use globally installed node.
-  download = true
+    // If true, it will download node using above parameters.
+    // If false, it will try to use globally installed node.
+    download = true
 
-  // Set the work directory for unpacking node
-  workDir = file("${project.buildDir}/nodejs")
+    // Set the work directory for unpacking node
+    workDir = file("${project.buildDir}/nodejs")
 
-  // Set the work directory where node_modules should be located
-  nodeModulesDir = file("${project.projectDir}")
+    // Set the work directory where node_modules should be located
+    nodeModulesDir = file("${project.projectDir}")
 }
 
 apply plugin: 'lesscss'
 apply plugin: 'dustjs'
 apply plugin: 'distribution'
 
 configurations {
-  generateRestli
+    generateRestli
 }
 
 dependencies {
-  compile project(':az-core')
-  compile(project(':azkaban-common'))
+    compile project(':az-core')
+    compile(project(':azkaban-common'))
+    compile(project(':az-flow-trigger-dependency-plugin'))
 
-  compile deps.restliServer
-  compile deps.velocityTools
+    compile deps.restliServer
+    compile deps.velocityTools
 
-  testCompile(project(path: ':azkaban-common', configuration: 'testCompile'))
-  testCompile deps.collections
+    testCompile(project(path: ':azkaban-common', configuration: 'testCompile'))
+    testCompile deps.collections
 
-  //AZ web module tests need to access classes defined in azkaban-common test module
-  testCompile project(':azkaban-common').sourceSets.test.output
+    //AZ web module tests need to access classes defined in azkaban-common test module
+    testCompile project(':azkaban-common').sourceSets.test.output
 
-  generateRestli deps.restliGenerator
-  generateRestli deps.restliTools
+    generateRestli deps.restliGenerator
+    generateRestli deps.restliTools
 
-  testCompile project(':test')
-  testCompile project(path: ':azkaban-db', configuration: 'testOutput')
-  testRuntime deps.h2
+    //testcompile project(':az-flow-trigger-dependency-plugin')
```

Comment:
I would believe az web server modules needs this test dependency if you write its implementations in the test.

Reply:
added testcompile